### PR TITLE
AppVeyor

### PR DESCRIPTION
**Pull request by ocefpaf**
@jswhit this is not ready yet. We need to,

- [x] create an AppVeyor account
- [x] activate AppVeyor on this repository
- [X] clean up the copy-and-paste of the `appveyor.yml` and recipe bolierplate
- [X] run the netcdf4-python test suit

Only you can do the first two as it requires adim privileges. I will take care of the last two once AppVeyor is enable.

### CODE REVIEW

**Comment by jswhit** [comment]
OK, first two on the list are done.

**Comment by ocefpaf** [comment]
@jswhit I closed and re-opened the PR to see if AppVeyor would kick-in, but nothing happened.

**Comment by jswhit** [comment]
I initiated an appveyor build manually, but it's trying to run on github master.  I don't see any way to tell it to run on a pull request.

**Comment by ocefpaf** [comment]
I have to confess that I set AppVeyor up only once and I don't recall all the switches to make it work.  Let me try closing and opening once more.  If that does not work I can re-send this PR.

**Comment by jhamman** [comment]
You may need to push another commit to this branch.

**Comment by ocefpaf** [comment]
Thanks @jhamman! Closing and opening does not work, but the extra commit did.

@jswhit even though AppVeyor is green (I did not expect that) I still want to polish this a little bit. I will ping you once I am done.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - TARGET_ARCH: x64
+      CONDA_NPY: 110
+      CONDA_PY: 35
+
+    - TARGET_ARCH: x86
+      CONDA_NPY: 19
+      CONDA_PY: 35
+    - TARGET_ARCH: x64
+      CONDA_NPY: 19
+      CONDA_PY: 35
```
Comment (ocefpaf) [inline appveyor.yml]: @jswhit AppVeyor is quite slow, so feel free to reduce this build matrix if you want. (Not sure if all that is needed.)

File: conda.recipe/run_test.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import os
+import netCDF4
+
+# Check OPeNDAP functionality.
+url = 'http://geoport-dev.whoi.edu/thredds/dodsC/estofs/atlantic'
+nc = netCDF4.Dataset(url)
+
```
Comment (ocefpaf) [inline conda.recipe/run_test.py]: The first two test above are *packaging tests* and may be removed. The test below is the actual unittests for `netCDF4-python`.

**Comment by jswhit** [comment]
I think we can get away with just testing np11+{x64,x86}+{py27,py35}.  I would like to keep the x86 test there, since we don't currently have any other tests for 32-bit architectures.

**Comment by ocefpaf** [comment]
> I think we can get away with just testing np11+{x64,x86}+{py27,py35}. 

Makes sense.  I just pushed another commit removing the unnecessary builds.

**Comment by jswhit** [comment]
OK, merging now.  Thanks @ocefpaf!

**Pull request merged**